Kickstarter-iOS/Views/Cells/ProjectPamphletMainCell.swift

```diff
@@ -84,6 +89,16 @@ internal final class ProjectPamphletMainCell: UITableViewCell, ValueCell {
   }
 
   internal func configureWith(value: (Project, RefTag?)) {
+    _ = ([self.creatorByLineView], self.projectNameAndCreatorStackView)
+      |> ksr_addArrangedSubviewsToStackView()
+
+    // todo: removing this
+    _ = self.creatorByLineView
+      |> \.isHidden .~ true
+
+//    _ = self.creatorStackView
+//      |> \.isHidden .~ true
+
```

Comment:
This is going to be removed in the next PR. To test, you should unhide the `creatorByLineView ` and uncomment the code that hides the `creatorStackView`